javascript/example-customer-account--order-action-menu--ts/extensions/action-menu-extension-menu-action-extension/locales/en.default.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Remember we dont want translations for the tutorial anymore so we can delete these (and use the strings directly) and the locales folder as well